lib/utils/logger.js

```diff
@@ -1,13 +1,26 @@
 const colors = require('colors');
 const {stackTraceParser} = require('./stackTraceParser');
-
+const {EOL} = require('os');
 const {config} = require('../../config');
 const logLevels = require('../constants/logLevels');
+const {SUITEST_LAUNCHER_PROCESS} = require('../constants/enviroment');
+
+/**
+ * Define custom colors
+ */
+const stColors = {
+	suit: colors.grey,
+	mild: colors.cyan,
+	bold: colors.cyan.bold,
+	errorColor: colors.red,
+	successColor: colors.green,
+};
 
 const methods = {
 	log: {
 		key: 'log',
 		color: 'white',
+		launcherColor: null,
```

Comment:
If null dont have any special meaning, I would prefer replace it ti empty string